### PR TITLE
Content fixes

### PR DESCRIPTION
**Pull request by kellrott**

### CODE REVIEW

File: content/explore.md [hunk 1]
```diff
@@ -3,61 +3,73 @@
...
+{{% card title="Ensembl" %}}
 The Ensembl project has been aggregating, processing, integrating and redistributing genomic datasets since the initial releases of the draft human genome, with the aim of accelerating genomics research through rapid open distribution of public data.
 <a href="https://academic.oup.com/nar/article/46/D1/D754/4634002" target="_blank">paper</a>
 {{% /card %}}
-{{% card title="tcga" %}}
- The Cancer Genome Atlas (TCGA) Research Network has profiled and analyzed large numbers of human tumours to discover molecular aberrations at the DNA, RNA, protein, and epigenetic levels. The resulting rich data provide a major opportunity to develop an integrated picture of commonalities, differences, and emergent themes across tumour lineages.
-<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3919969/" target="_blank">paper</a>
+
+{{% card title="PFAM" %}}
+<a href="https://pfam.xfam.org/" target="_blank">PFAM</a> describes over 13,000 protein families (<a href="https://www.ncbi.nlm.nih.gov/pubmed/22127870" target="_blank">paper</a>)
```
Comment (bwalsh) [inline content/explore.md]: why is this (paper) ?

File: content/explore.md [hunk 2]
```diff
@@ -3,61 +3,73 @@
...
+
+{{% card title="MC3" %}}
 The Cancer Genome Atlas (TCGA) cancer genomics dataset includes over 10,000 tumor-normal exome pairs across 33 different cancer types, in total >400 TB of raw data files requiring analysis.
 <a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(18)30096-6" target="_blank">paper</a>
 {{% /card %}}
-{{% card title="ensembl" %}}
+
+{{% card title="CTDD" %}}
+Cancer Therapeutics Response Portal (CTRP) catalogues response profiles of 481 compounds against 860 cancer cell lines.
+{{% /card %}}
```
Comment (bwalsh) [inline content/explore.md]: why no paper ?

File: content/explore.md [hunk 3]
```diff
@@ -3,61 +3,73 @@
...
+{{% card title="GO" %}}
+Gene Ontology Consortium is a controlled vocabulary describing knowledge of gene and protein roles in cells.
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/10802651" target="_blank">paper</a>
 {{% /card %}}
-{{% card title="gtex" %}}
-Understanding the functional consequences of genetic variation, and how it affects complex human disease and quantitative traits, remains a critical challenge for biomedicine. We present an analysis of RNA sequencing data from 1641 samples across 43 tissues from 175 individuals, generated as part of the pilot phase of the Genotype-Tissue Expression (GTEx) project. We describe the landscape of gene expression across tissues, catalog thousands of tissue-specific and shared regulatory expression quantitative trait loci (eQTL) variants, describe complex network relationships, and identify signals from genome-wide association studies explained by eQTLs.
-<a href="http://science.sciencemag.org/content/348/6235/648" target="_blank">paper</a>
+
+{{% card title="PubMed" %}}
+Publication information from over X articles
```
Comment (bwalsh) [inline content/explore.md]: Can we define X

**Comment by adamstruck** [comment]
`/queries/matrix` returns 404

**Comment by adamstruck** [comment]
@bwalsh can you check that your requested changes were made? Otherwise this PR looks ready to merge to me.

**Comment by bwalsh** [comment]
@kellrott @adamstruck  can you review per today's conversation?
https://bmeg-staging.ddns.net/analyze/getting_started/

**Pull request merged**